docker-compose.yml

```diff
@@ -12,17 +12,13 @@ services:
       - "8888"
     volumes:
       - nodeos-data-volume:/opt/eosio/bin/data-dir
-    environment:
-      - VIRTUAL_HOST=local.eosio.cr
-      - VIRTUAL_PORT=8888
-    cap_add:
-      - IPC_LOCK
-    stop_grace_period: 10m
     networks:
       eos-local:
         aliases:
           - local.eosio.cr
-
+    environment:
+      - VIRTUAL_HOST=local.eosio
```

Comment:
Virtual Host and Network aliases must match so that you can access the service both via ngnix reverse proxy from the host machine and within the docker network with the same url.

Reply:
Done!